test/e2e/specs/published-content/likes__comment.ts

```diff
@@ -53,10 +53,31 @@ describe( 'Likes: Comment', function () {
 			}
 		);
 
-		newComment = await restAPIClient.createComment(
+		commentToBeLiked = await restAPIClient.createComment(
+			testAccount.credentials.testSites?.primary.id as number,
+			newPost.ID,
+			DataHelper.getRandomPhrase()
+		);
+		commentToBeUnliked = await restAPIClient.createComment(
 			testAccount.credentials.testSites?.primary.id as number,
 			newPost.ID,
-			comment
+			DataHelper.getRandomPhrase()
+		);
+
+		// For AT sites the API will respond with a HTTP 404
+		// unless time is given for the comment to "settle" in place.
+		// It could be argued that adding this arbitrary delay is
+		// "more representative" of users.
+		// @see: https://github.com/Automattic/wp-calypso/issues/75952
+		if ( envVariables.TEST_ON_ATOMIC ) {
+			await page.waitForTimeout( 5 * 1000 );
```

Comment:
As explained in the code, this is not an ideal situation, but one I could reproduce repeatedly in my local testing.

With a shorter or non-existent timeout here, the rate of failure for this spec in AT environment is very high - not 100%, but more failures than not.